srcup/api.py

```diff
@@ -29,6 +29,12 @@ class Config:
         headers={"x-api-key": api_key}, json_serialize=lambda x: x.json()
     ) as session:
         print("Uploading...")
+
+        project_id = await get_project_id(watchdog_api, api_key, name)
+
+        if project_id != -1:
+            raise Exception(f"Project with name {name} already exists")
```

Comment:
If we are to return `None` (see comment below) this will need to be an `is None` check.

Reply:
Done

Reply:
I don't t see the `is not None` check